src/test/java/com/epam/automation/inside.java

```diff
@@ -1,89 +1,129 @@
+    private String city = "";
+    @FindBy(xpath = "//*[contains(@src, 'logo_white-blue.svg')]")
+    private WebElement logo;
+    @FindBy(xpath = "//*[starts-with(@id,'select-box-location-')]")
+    private WebElement locationArrow;
+    @FindBy (xpath = "//*[contains(@id, 'all_locations')]")
+    private WebElement defaultLocation;
+    @FindBy (xpath = "//*[starts-with(@class, 'job-search__input')]")
+    private WebElement keywordInput;
+    @FindBy (xpath =  "/html/body/div[1]/div[3]/div[1]/div[1]/section/div/div[4]/form/div/div[2]/div/div[1]/div[1]")
+    private WebElement skillsTabArrow;
+    @FindBy (xpath = "/html/body/div[1]/div[3]/div[1]/div[1]/section/div/div[4]/form/div/div[3]/button")
+    private WebElement searchButton;
+    @FindBy (className = "search-result__item-name")
+    private WebElement searchResult;
+    @FindBy (css = "li.search-result__sorting-item:nth-child(2)")
+    private WebElement sortByDate;
+
+    @BeforeClass
     public void openBrowser() {
         System.setProperty("webdriver.gecko.driver", "drivers/geckodriver.exe");
         driver = new FirefoxDriver();
         driver.get("https://www.epam.com");
-        System.out.println(driver.getTitle());
-        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        log.info("Epam website opened");
+        PageFactory.initElements(driver, this);
     }
 
     public void navigation(String Menu) {
-        // menüpontokra tud lépni enumból kiszedett címekkel, almenüket jelenleg nem raktam bele
-        // az oldal sikeres betöltését azzal ellenőrzi, hogy megjelent e a bal felső sarokban lévő epam logó
-        driver.findElement(By.xpath("//a[@href='" + Menu + "']")).click();
-        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
-        System.out.println(driver.getTitle());
-        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
-        assertTrue(driver.findElement(By.xpath("//*[contains(@src, 'logo_white-blue.svg')]")).isDisplayed());
-        /* Updatelve lett az Epam oldal, ami most már az országot és a várost automatikusan kitölti az általad szedett adatokból
-        ezért előre beállítom, hogy mindig állítsa vissz alapértékre a locationt az egyszerűség kedvéért
+        /**
+         * Navigation between main menus
+         * @param Menu getting a String from enums which refers to a main menu
+         * Checking the site load correctly with the logo display
+         * with the latest Epam website update we always set the location back to All location by default
+         * since its get the location from our ip address
          */
-        if (Menu.equals(com.epam.automation.enums.Menu.CAREER)) {
-            driver.findElement(By.xpath("//*[starts-with(@id,'select-box-location-')]")).click();
-            driver.findElement(By.xpath("//*[contains(@id, 'all_locations')]")).click();
+        driver.findElement(By.xpath("//a[@href='" + Menu + "']")).click();
```

Comment:
you have webelement objects now, you don't need to use method like this.

Reply:
I didn't really find a way to use it as an object because its uses Menu String which is dynamic